docs/features/autoupdater.rst

```diff
@@ -3,6 +3,13 @@ Autoupdater
 
 Gluon contains an automatic update system which can be configured in the site configuration.
 
+Please mind, that asking users for consent is not implemented within the automatic update system.
+Also, users are neither asked nor notified when updates are available or installed.
+
+It appears reasonable to ask users for consent before administrators gain access to private devices and networks via automatic updates. This could be done before downloading, e.g. by providing different download options.
```

Comment:
That sounds very vague ("it appears that...") and because it is a serious matter it should not.

Reply:
The effort to provide a dedicated set of images with a disabled autoupdater feels disproportionate. I'd much prefer a in-firmware solution.